packages/nx-plugin/src/executors/autorun/Readme.md

```diff
@@ -0,0 +1,60 @@
+# Autorun Executor
+
+This executor is used to run the code-pushup CLI autorun command in a Nx workspace.
+For details on the CLI command read the [CLI autorun documentation](https://github.com/code-pushup/cli/blob/main/packages/cli/README.md#autorun-command).
+
+#### @code-pushup/nx-plugin:autorun
+
+## Usage
+
+// project.json
+
+```json
+{
+  "name": "my-project",
+  "targets": {
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun"
+    }
+  }
+}
+```
+
+By default, the Nx plugin will derive the options from the executor config.
+
+The following things happen:
+
+- the output directory defaults to `${workspaceRoot}/.code-pushup/${projectName}`
+- the config file defaults to `${projectRoot}/code-pushup.config.ts`
+- parses terminal arguments and forwards them to the CLI command (they override the executor config)
+- the CLI command is executed
+
+```jsonc
+{
+  "name": "my-project",
+  "targets": {
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun",
+      "options": {
+        "projectPrefix": "my-project",
```

Comment:
We already know what the project `name` is, so how about making it a boolean flag? Then the name doesn't have to duplicated.

```suggestion
        "projectPrefix": true,
```

It's suggestion, though, and doesn't have to be in scope of this PR.

Reply:
Srythis was a typo. Makes no sense. I fixed the typo and added comments.